backend/src/main/java/com/bakdata/conquery/models/worker/Worker.java

```diff
@@ -34,12 +34,12 @@
 	@Getter
 	private final ExecutorService executorService;
 	
-	public Worker(WorkerInformation info, JobManager jobManager, WorkerStorage storage, QueryExecutor queryExecutor, ExecutorService executorService) {
+	public Worker(WorkerInformation info, JobManager jobManager, WorkerStorage storage, QueryExecutor queryExecutor, ExecutorService executorService, int entityBucketSize) {
```

Comment:
`entityBucketSize` sollte in der `WorkerInformation` stecken, das wird auf dem release-Branch deutlicher. Dort existiert die WorkerInformation nur noch im WorkerStorage und bleibt so konsistent für den Worker sollte sich die entityBucketSize zwischen neustarts ändern.